Content.Client/RoundEnd/RoundEndSummaryUIController.cs

```diff
@@ -16,6 +19,17 @@ public sealed class RoundEndSummaryUIController : UIController,
     [Dependency] private readonly IInputManager _input = default!;
 
     private RoundEndSummaryWindow? _window;
+    private MenuButton? ToggleRoundEndSummaryWindowButton => UIManager.GetActiveUIWidgetOrNull
+        <UserInterface.Systems.MenuBar.Widgets.GameTopMenuBar>()?.ToggleRoundEndSummaryWindowButton;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        var gameplayStateLoad = UIManager.GetUIController<GameplayStateLoadController>();
+        gameplayStateLoad.OnScreenLoad += LoadButton;
+        gameplayStateLoad.OnScreenUnload += UnloadButton;
+    }
```

Comment:
This is being done in reverse, the button itself should be trying to toggle this controller and not vice versa.